megiddo/utils_table.h

```diff
@@ -10,7 +10,7 @@ void afficherTableauPoint(POINT tab[],int taille);
 void afficherTableauDoublet(DOUBLET tab[],int taille);
 void echangerDoublet(DOUBLET tableau[], int a, int b);
 void echangerPoint(POINT tableau[], int a, int b);
-void quickSortDoublet(DOUBLET tableau[], int debut, int fin);
+double quickSortDoublet(DOUBLET tableau[], int debut, int fin);
```

Comment:
il y a un warning a cause de ce changement: 
utils_table.c:96:1: warning: control reaches end of non-void function [-Wreturn-type]
 }